bpf/lib/trace.h

```diff
@@ -143,112 +143,87 @@ static __always_inline void
 send_trace_notify(struct __ctx_buff *ctx, __u8 obs_point, __u32 src, __u32 dst,
 		   __u16 dst_id, __u32 ifindex, __u8 reason, __u32 monitor)
 {
+	__u64 ctx_len = ctx_full_len(ctx);
+	__u64 cap_len = min_t(__u64, !monitor ? TRACE_PAYLOAD_LEN : monitor,
```

Comment:
Nit: `!monitor ? TRACE_PAYLOAD_LEN : monitor` is the same as `monitor ? : TRACE_PAYLOAD_LEN` I think, which might be more readable?

Reply:
Ah yeah of course, will fix :-)